cplusplus/remove_duplicates.cpp

```diff
@@ -0,0 +1,12 @@
+#include <bits/stdc++.h>
+#include <iostream>
+
+int main()
+{
+    // Expected return of [1, 4, 2, 50, 3, 70, 14]
+    int duplicates[10] = {1, 4, 2, 1, 50, 50, 3, 2, 70, 14};
+    std::set<int> nonDuplicates;
+    std::set<int>::iterator it;
+    //Spaghetti code BUT it is one line
+    for (int i = 0; i < 10; i++) nonDuplicates.insert(duplicates[i]); for (it = nonDuplicates.begin(); it != nonDuplicates.end(); it++) std::cout << *it << ", ";
```

Comment:
Since this code actually removes the duplicates:

> for (int i = 0; i < 10; i++) nonDuplicates.insert(duplicates[i]);

this counts. The second half separated by the `...[i]); for (it...` technically is optional logging.

Reply:
Ah, I was under the impression that you wanted it logged too. By the way this is supposed to be the JavaScript version, but because the C++ PR is still open, this is here.